applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -28,6 +28,10 @@ def _ValidateSettings(self, settings):
             "formulation": {
                 "element_type": "imcompressible_potential"
             },
+            "element_replace_settings": {
```

Comment:
Why do you modify this?

Reply:
otherwise it is not reading this statement from the Parameters file hence having an error while compiling

Reply:
This is definitively not the way to go. The element and condition types must be set using the `formulation` fields on top, as is done in the FluidDynamicsApplication. The problem is in your .json file, not in the solver.